packages/libs/src/sdk/api/generated/full/models/Purchase.ts

```diff
@@ -83,6 +83,12 @@ export interface Purchase {
      * @memberof Purchase
      */
     updatedAt: string;
+    /**
+     * 
+     * @type {string}
```

Comment:
I wonder if we have a way to actually output these as enums? Don't have to solve for this PR, but it's worth exploring at some point.